app/main.py

```diff
@@ -2,17 +2,20 @@
 from fastapi import FastAPI, Request
 from fastapi.responses import JSONResponse
 from fastapi_jwt_auth.exceptions import AuthJWTException
```

Comment:
this is not working for me on local
can you please check what's the issue?

Reply:
Not getting any error on local for me.
<img width="887" alt="Screenshot 2022-12-01 at 9 43 23 AM" src="https://user-images.githubusercontent.com/25496389/204964174-fb8d3f29-af82-4cc2-9b28-89da0c7d27c7.png">